tests/test_repository.py

```diff
@@ -9,11 +9,16 @@
 from pytest_container.runtime import LOCALHOST
 
 from bci_tester.data import BASE_CONTAINER
+from bci_tester.data import BCI_REPO_NAME
 from bci_tester.data import OS_SP_VERSION
 from bci_tester.data import OS_VERSION
 from bci_tester.data import REPOCLOSURE_CONTAINER
 
 
+_RM_ZYPPSERVICE = (
+    "rm -v /usr/lib/zypp/plugins/services/container-suseconnect-zypp"
```

Comment:
@grisu48 because otherwise the repository tests are using all repositories (e.g. the ones from the host system that was registered against SCC) rather than just the SLE_BCI repository for testing (which is the intention of the tests as we actually want to use it to gate whether we can erlease a new SLE BCI repository). the previous tests have been ineffective because they actually tested the customer repositories instead.

Reply:
I moved this part into #334